package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hashgraph/smart-contracts",
-  "version": "0.5.0-SNAPSHOT",
+  "version": "0.5.0",
```

Comment:
```suggestion
  "version": "0.6.0-SNAPSHOT",
```

Reply:
Apologies, I thought I set the target branch to release/0.5.0.  Corrected now.